components/markets/market-card/index.tsx

```diff
@@ -45,26 +47,63 @@ const Pill = ({ value, classes }: { value: string; classes: string }) => {
   );
 };
 
-const MarketCardInfo = ({ question }: { question: string }) => {
+const MarketCardInfo = ({
+  question,
+  img,
+}: {
+  question: string;
+  img: string;
+}) => {
   return (
-    <div className="w-full h-full flex flex-col text-ztg-14-165 whitespace-normal">
-      <h5 className="font-semibold w-full h-fit line-clamp-3">{question}</h5>
+    <div className="w-full h-full flex whitespace-normal gap-4">
+      <h5 className="font-semibold w-full h-fit line-clamp-3 text-base">
+        {question}
+      </h5>
+      {img && <MarketImage image={img} alt={question} className="rounded-lg" />}
     </div>
   );
 };
 
-const MarketCardTags = ({ tags }: { tags: string[] }) => {
+const MarketCardTags = ({
+  tags,
+  baseAsset,
+  isVerified,
+}: {
+  tags: string[];
+  baseAsset: string;
+  isVerified: boolean;
+}) => {
   return (
     <>
-      {tags?.map((tag, index) => {
+      {/* update later to include other assets */}
```

Comment:
Could you make this dynamic? Should be able to to if with `lookupAssetImagePath` or something similar